pyuvsim/simsetup.py

```diff
@@ -369,24 +574,24 @@ def initialize_catalog_from_params(obs_params, input_uv=None):
 
         time = mock_keywords.pop('time')
 
-        catalog, mock_keywords = create_mock_catalog(time, return_table=True, **mock_keywords)
+        sky, mock_keywords = create_mock_catalog(time, **mock_keywords)
         mock_keyvals = [str(key) + str(val) for key, val in mock_keywords.items()]
         source_list_name = 'mock_' + "_".join(mock_keyvals)
     elif isinstance(catalog, str):
         source_list_name = os.path.basename(catalog)
         if not os.path.isfile(catalog):
             catalog = os.path.join(param_dict['config_path'], catalog)
         if catalog.endswith("txt"):
-            catalog = pyradiosky.read_text_catalog(catalog, return_table=True)
+            sky = pyradiosky.read_text_catalog(catalog)
```

Comment:
The function you're calling here is deprecated. Can you use `SkyModel.read_text_catalog`?

Reply:
updated!